lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.t.d/svs.hq.sv.NA12878.20slice.30X2.aligned.bam.readdepth.bed

```diff
@@ -0,0 +1,2 @@
+20_slice	51001	53900	0.559811
+20_slice	260701	263200	0.504321
```

Comment:
Are these `30X2` files being used anywhere?  At first glance they're identical to the `30X` versions.